packages/react/src/component.tsx

```diff
@@ -3,13 +3,14 @@ import { useCallback, useRef, useState } from "react";
 import type { FileWithPath } from "react-dropzone";
 import { useDropzone } from "react-dropzone";
 
+import { ExpandedRouteConfig } from "@uploadthing/shared";
```

Comment:
This should still be a type import, no?

Reply:
Yep - I'm not sure why the linter isn't yelling 👀